src/user/model.js

```diff
@@ -0,0 +1,32 @@
+// Use sequelize db connection 
+const { DataTypes } = require("sequelize");
+const { sequelize } = require("../db/connection");
+
+
+const User =sequelize.define("User", 
+  {
+    id:{
```

Comment:
please remove id field from model. This will be set automatically by the DB provider.